docs/howtos/exposing-rust-components-to-kotlin.md

```diff
@@ -144,3 +144,24 @@ However, it comes with the following downsides:
    [`ffi_support` docs](https://docs.rs/ffi-support/*/ffi_support/), but a
    major one is when to use `Pointer` vs `String` (getting this wrong will
    often work, but may corrupt memory).
+
+### How do I debug Rust code with the step-debugger in Android Studio
+
+1. Uncomment the `packagingOptions { doNotStrip "**/*.so" }` line from the
+   build.gradle file of the component you want to debug.
+2. In the rust code, either:
```

Comment:
This should not be necessary.  You want to use the dual debugger.  Then you use the Java debugger to break at a reasonable point (say, megazord init, immediately after the library loads), at which point you configure the native debugger.  Assuming you have symbols, you'll be able to use the LLDB console.  Then continue in the Java debugger.

See https://mozilla.github.io/geckoview/tutorials/native-debugging.html#attaching-debuggers-to-content-and-other-child-processes for the equivalent, with some process fun thrown in.